src/cli/mod.rs

```diff
@@ -340,6 +352,7 @@ impl ReplCli {
             &self.public_params_dir,
             &self.proofs_dir,
             &self.commits_dir,
+            &None, // TODO: help
```

Comment:
@samuelburnham I ran into these changes when rebasing. For the sake of compiling, I put None here, but this is probably not correct. Should `circom` configuration files go with `LURK_DIR`s? I'm not sure what the final structures here will look like; what should be done?

Reply:
As mentioned in DMs this could be `&self.circom_dir` and have the same behavior as the other `LURK_DIRS`

Reply:
Done, hopefully this matches the behavior of the other directories, I've requested a review